satpy/composites/__init__.py

```diff
@@ -668,11 +668,16 @@ def __init__(self, name, lim_low=85., lim_high=88., day_night="day_night", **kwa
             day_night (string): "day_night" means both day and night portions will be kept
                                 "day_only" means only day portion will be kept
                                 "night_only" means only night portion will be kept
+            include_alpha (bool): This only affects the "day only" or "night only" result.
+                                  True means an alpha band will be added to the output image for transparency.
+                                  False means the output is a single-band image with undesired pixels being masked out
+                                   (replaced with NaNs).
```

Comment:
Documentation is failing with:

```
/lib/python3.10/site-packages/satpy/composites/__init__.py:docstring of satpy.composites.DayNightCompositor:22: ERROR: Unexpected indentation.
```

I think this should fix it:

```suggestion
            include_alpha (bool): This only affects the "day only" or "night only" result.
                                  True means an alpha band will be added to the output image for transparency.
                                  False means the output is a single-band image with undesired pixels being masked out
                                  (replaced with NaNs).
```